CHANGELOG.md

```diff
@@ -1,5 +1,118 @@
+- Move `address` field on CensusEntry to `sys.gossip\_ip` [\#1761](https://github.com/habitat-sh/habitat/pull/1761) ([reset](https://github.com/reset))
+- Add `pkg\_binds` and `pkg\_binds\_optional` plan variables [\#1760](https://github.com/habitat-sh/habitat/pull/1760) ([reset](https://github.com/reset))
+- update Appveyor's rust to 1.15.1 stable [\#1758](https://github.com/habitat-sh/habitat/pull/1758) ([smurawski](https://github.com/smurawski))
+- Disambiguate key 'port' from listening port [\#1747](https://github.com/habitat-sh/habitat/pull/1747) ([davidwrede](https://github.com/davidwrede))
+- Build devshell in docker-compose [\#1746](https://github.com/habitat-sh/habitat/pull/1746) ([bodymindarts](https://github.com/bodymindarts))
+- Fix typo in text [\#1744](https://github.com/habitat-sh/habitat/pull/1744) ([tashimi](https://github.com/tashimi))
+- Refactor Supervisor main loop to tick over each Service on each tick [\#1734](https://github.com/habitat-sh/habitat/pull/1734) ([reset](https://github.com/reset))
+- \[sup\] Refactor most global `Config` into `ManagerConfig` & `ServiceSpec` [\#1732](https://github.com/habitat-sh/habitat/pull/1732) ([fnichol](https://github.com/fnichol))
+- Add Smoke Test hook [\#1725](https://github.com/habitat-sh/habitat/pull/1725) ([reset](https://github.com/reset))
+- Package activity should be properly rooted in a windows studio [\#1724](https://github.com/habitat-sh/habitat/pull/1724) ([mwrock](https://github.com/mwrock))
+- Begin Supervisor CLI parsing refactor [\#1722](https://github.com/habitat-sh/habitat/pull/1722) ([fnichol](https://github.com/fnichol))
+- Fix pkg\_exports syntax errors in tutorial and builder plans [\#1720](https://github.com/habitat-sh/habitat/pull/1720) ([reset](https://github.com/reset))
+- revert change to build.ps1 [\#1715](https://github.com/habitat-sh/habitat/pull/1715) ([smurawski](https://github.com/smurawski))
+-  Rename pkg\_expose to pkg\_exposes and refactor to leverage pkg\_exports [\#1712](https://github.com/habitat-sh/habitat/pull/1712) ([reset](https://github.com/reset))
+- Update Building.md for Windows to match our current process [\#1709](https://github.com/habitat-sh/habitat/pull/1709) ([smurawski](https://github.com/smurawski))
+- Depot validates hab client target \(platform/architecture\) before returning packages [\#1707](https://github.com/habitat-sh/habitat/pull/1707) ([smurawski](https://github.com/smurawski))
+- \[hab\] Install same version of helper packages as hab if not present. [\#1700](https://github.com/habitat-sh/habitat/pull/1700) ([fnichol](https://github.com/fnichol))
+- Document the log keys [\#1699](https://github.com/habitat-sh/habitat/pull/1699) ([smith](https://github.com/smith))
+- \[hab-spider\] Remove release profile from Cargo.toml. [\#1696](https://github.com/habitat-sh/habitat/pull/1696) ([fnichol](https://github.com/fnichol))
```

Comment:
@chefsalim You'll want to label them that way with github labels, otherwise the next gen of the release will add them back.